### DISCUSSION TITLE
Collaboration

### DISCUSSION THREAD

**Discussion by oorosh** [tryAGI/LangChain#2] [General]
Hi @HavenDV , 

I stumbled upon this repo, and we work on the same thing in parallel :-)

checkout this repo, where im contributor. We have 3 devs working on it.

https://github.com/jeastham1993/langchain-dotnet

It would be great if we could collaborate. 

cheers

**Comment by HavenDV** [comment]
Hi @oorosh.

Yes, I don't mind cooperating. But for me, the main issue of running a long-term Open-Source project is the availability of direct or indirect (for example, you can do this as part of your work) funding. And I would like to know, on the basis of what are you currently leading the project?
At the moment, my project is developing as a side project while working on the main product and pursuing its goals. Therefore, for example, I focus on OpenAI functions and the simplest work with them.
